spec/active_record/connection_adapters/oracle_enhanced_data_types_spec.rb

```diff
@@ -882,6 +882,234 @@ class ::TestSerializeEmployee < ActiveRecord::Base
   end
 end
 
+describe "OracleEnhancedAdapter handling of NCLOB columns" do
+  include SchemaSpecHelper
+
+  before(:all) do
+    ActiveRecord::Base.establish_connection(CONNECTION_PARAMS)
+    @conn = ActiveRecord::Base.connection
+    schema_define do
+      create_table :test_employees, force: true do |t|
+        t.string  :first_name, limit: 20
+        t.string  :last_name, limit: 25
+        t.column  :comments, :ntext
+      end
+      create_table :test2_employees, force: true do |t|
+        t.string  :first_name, limit: 20
+        t.string  :last_name, limit: 25
+        t.column  :comments, :ntext
+      end
+      create_table :test_serialize_employees, force: true do |t|
+        t.string  :first_name, limit: 20
+        t.string  :last_name, limit: 25
+      end
+      add_column :test_serialize_employees, :comments, :ntext
+    end
+
+    @nchar_data = "日オメヌ"
+    @nchar_data2 = "使セ転"
+
```

Comment:
What do these two `nchar_data` and `nchar_data2` represent?  These values look meaningless and garbled for me. Note : I speak Japanese and English.